src/applications/personalization/profile360/util/index.js

```diff
@@ -43,6 +46,12 @@ const hasErrorMessage = (errors, errorKey, errorText) => {
   );
 };
 
+export const hasAccountFlaggedError = errors =>
+  hasErrorMessage(errors, ACCOUNT_FLAGGED_FOR_FRAUD_KEY);
+
+export const hasRoutingNumberFlaggedError = errors =>
+  hasErrorMessage(errors, ROUTING_NUMBER_FLAGGED_FOR_FRAUD_KEY);
+
 export const hasFlaggedForFraudError = errors =>
```

Comment:
we still need this old function because we use it to determine which error message to show the user. But that will eventually change as well (we'll show different errors depending on which error code we see: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4109).